examples/getstarted/config/plugins.js

```diff
@@ -13,4 +13,11 @@ module.exports = ({ env }) => ({
       },
     },
   },
+  myplugin: {
+    enabled: true,
+    resolve: `${__dirname}/../plugins/myplugin`,
```

Comment:
We can avoid to use `__dirname` by prefixing the path with `strapi.dir`. What do you think?
If yes, do you want to keep the ability to declare the package name instead of the path ?
Today, if you did `yarn add my-npm-package`, you can load it like that:
```js
resolve: 'my-npm-plugin',
```
However this feature is maybe useless. So, would we keep it or not?

Reply:
I know it's an example but when dealing with path you can use `path`.
In this case you can do something like `path.resolve(__dirname, '../plugins/myplugin')` or `path.resolve(strapi.dir, '../plugins/myplugin')`.

Regarding your question, I like the `strapi.dir` idea. It removes "complexity" from the end user. It's more straight forward (and probably easier to document). 

If using `strapi.dir` makes declaring a plugin using the package name obsolete in favor of using the package path, I would vote for removing that possibility as it's no longer needed. 

Just take my comments with a grain of salt since I don't have lots of insight in the code base and inner working at the moment.

Reply:
I agree with what you said @Elhebert !

Reply:
These are two different usage complementary to each other.

One is to load a locale plugin that's not published on npm and the other is just to resolve a pacakge by name. 
Not allowing a simple package name to force doing `path.join(strapi.dir , 'node_modules/..../....')` is a bad solution. Node module resolution is a lot more powerfull and does tree search to find the module to allow more architectures like monorepos where some plugins might be installed in a parent directory